packages/plugins/custom-fields/admin/src/components/ColorPicker/ColorPickerInput/index.js

```diff
@@ -0,0 +1,178 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Stack } from '@strapi/design-system/Stack';
+import { Typography } from '@strapi/design-system/Typography';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { BaseButton } from '@strapi/design-system/BaseButton';
+import { Field, FieldHint, FieldError, FieldLabel, FieldInput } from '@strapi/design-system/Field';
+import CarretDown from '@strapi/icons/CarretDown';
+import { useIntl } from 'react-intl';
+import { HexColorPicker } from 'react-colorful';
+import useOnClickOutside from '../../../hooks/useOnClickOutside';
+import getTrad from '../../../utils/getTrad';
+
+const ColorPreview = styled.div`
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  margin-right: 10px;
+  background-color: ${(props) => props.color};
+  border: 1px solid rgba(0, 0, 0, 0.1);
```

Comment:
Are we sure we want another shade of grey?

Reply:
@maevalienard is there a value from [the DS colors](https://github.com/strapi/design-system/blob/main/packages/strapi-design-system/src/themes/colors.js) I could use?